bika/lims/browser/worksheet/workflow.py

```diff
@@ -109,7 +109,7 @@ def __call__(self):
                 # consistent with the order of the ARs
                 catalog = get_tool(CATALOG_ANALYSIS_LISTING)
                 brains = catalog({'UID': analysis_uids,
-                                 'sort_on': 'getRequestID'})
+                                 'sort_on': 'sortable_title'})
```

Comment:
Wrong. this should be `getRequestID`. Otherwise, the manage_results view from worksheet will appear wrong, like described in https://github.com/senaite/bika.lims/issues/302 . We only sort analyses here for adding them in the worksheet, not for displaying them in a list.
The reason is that we need the Analyses to be sorted in AR blocks (cause there is some logic there that mix them otherwise). Also AR slots need to be sorted with the same natural order as were registered.